scripts/validate-deployment.sh

```diff
@@ -5,19 +5,20 @@ set -euo pipefail
 ## Query Prometheus for the daml_health_status metric and validate that it has the value 1, representing a healthy service
 
 retry=0
-result=
-until [ "$retry" -ge 5 ]; do
+result=''
+
+until [[ "${retry}" -ge 5 ]]; do
   # Query Prometheus for daml_health_status >= 1
-  result=$(curl --retry 10 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
+  result=$(curl -fsS --retry 5 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
```

Comment:
Can you expand a bit as to why we changed the curl flags and the number of retries?

Reply:
* No output from servers failure or curl
* Show only curl errors
* Curl retry is exponential by default without `--retry-delay <seconds>` , reaching 10 retries will take a total time off ~17 minutes (default delay in seconds: 1, 2, 4, 8, 16, 32, 64, 128, 256, 512), I think it's too much. We could also set `--retry-delay 5` for example.